README.md

```diff
@@ -11,9 +11,9 @@
 
 # Get started
 
-Install it via: `brew install kubesphere-sigs/kubesphere-sigs/ks`
+Install it via: `brew install linuxsuren/linuxsuren/hd`
```

Comment:
This instruction should be able to install `ks` instead of `hd`.

See also https://github.com/LinuxSuRen/homebrew-linuxsuren/blob/master/Formula/ks.rb